packages/web-components/src/global/_variables.scss

```diff
@@ -32,13 +32,15 @@ $color-gray-warm-dark: #494440;
 $color-gray-warm-light: #e4e2e0;
 $color-gray-cool-light: #dce4ef;
 $v3-color-error-dark: #b50909;
-$v3-color-base-darkest: #1B1B1B;
-$font-family-sans: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif;
+$v3-color-base-darkest: #1b1b1b;
+$font-family-sans: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Roboto, Arial,
+  sans-serif;
 $font-family-serif: Bitter, Georgia, Cambria, 'Times New Roman', Times, serif;
 $font-weight-normal: 400;
 $font-weight-bold: 700;
 $font-style-normal: normal;
 $font-style-italic: italic;
+$base-v3-font-size: 16px;
```

Comment:
This file is generated dynamically by the css-library (I forgot about this in previous reviews) so it has been updated in this PR already:

https://github.com/department-of-veterans-affairs/component-library/pull/624#discussion_r1119376756

Can changes here and in `_functions.scss` be reverted in this PR? Those updates are in `main` already.